File: BotsSDK/korebot/src/main/java/kore/botssdk/audiocodes/webrtcclient/Permissions/PermissionManagerType.java
package kore.botssdk.audiocodes.webrtcclient.Permissions;


public enum PermissionManagerType
{
    CONTACTS(PermissionWrapper.READ_CONTACTS),
    PHONE(PermissionWrapper.READ_PHONE_STATE),
    CAMERA(PermissionWrapper.CAMERA),
    MICROPHONE(PermissionWrapper.RECORD_AUDIO);

    private String typeName;

    PermissionManagerType(String name)
    {
        this.typeName = name;
    }

    public String getTypeName()
    {
        return typeName;
    }
}

File: BotSdk/ui/src/main/java/com/kore/ui/audiocodes/webrtcclient/permissions/PermissionManagerType.kt
package com.kore.ui.audiocodes.webrtcclient.permissions

enum class PermissionManagerType(@JvmField val typeName: String) {
    CONTACTS(PermissionWrapper.READ_CONTACTS),
    PHONE(PermissionWrapper.READ_PHONE_STATE),
    CAMERA(PermissionWrapper.CAMERA),
    MICROPHONE(PermissionWrapper.RECORD_AUDIO)

}
